### PR TITLE
hashed password field, added fixtures, updated and added more fields …

### PR DESCRIPTION
**Pull request by ethanmnrd**
Changes made in this PR:

1. Reconfigured the api pathing for the endpoints.
2. Updated README.md to the most current backend steps.
3. Password hashing upon user creation (through custom User model) should now work.
4. Fixtures have been added, but must be edited in more depth.
5. Migrated database from sqlite to new mysql database.
6. Added steps to initialize database and migrate models to tables.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -45,3 +45,4 @@ flow-typed/npm/*
 .idea
 npm-debug.log.*
 __snapshots__
+.vscode/settings.json
```
Comment (vikr01) [inline .gitignore]: I actually would not add this to `.gitignore` -- it's helpful for other contributors using VSCode.

File: .vscode/settings.json [hunk 2]
```diff
@@ -1,3 +1,3 @@
 {
-    "python.pythonPath": "C:\\Users\\Ethan\\.virtualenvs\\cs160-j7hg6mN6\\Scripts\\python.exe"
+    "python.pythonPath": "C:\\Users\\ddoem\\.virtualenvs\\projectdev-TXkBElKe\\Scripts\\python.exe"
 }
```
Comment (jooohhn) [inline .vscode/settings.json]: Fix new line

File: server/etes/admin.py [hunk 3]
```diff
@@ -1,3 +1,7 @@
 from django.contrib import admin
+from .models import User, Ticket, Event
 
 # Register your models here.
+admin.site.register(User)
+admin.site.register(Ticket)
+admin.site.register(Event)
```
Comment (jooohhn) [inline server/etes/admin.py]: Fix new line

**Comment by jooohhn** [comment]
- Summarize major changes done in this PR more specifically
- Update README with the most important information (e.g. what is the API)

File: .vscode/settings.json (hunk 2 above)
Comment (vikr01) [reply .vscode/settings.json]: This was added to `.gitignore` but not added to the repo. @ethanmnrd should use `git rm --cached .vscode/settings.json` if you don't want it to be committed to the repo anymore.

File: server/etes/admin.py (hunk 3 above)
Comment (vikr01) [reply server/etes/admin.py]: I recommend adding `"files.insertFinalNewline": true` to `.vscode/settings.json` so VSCode will handle this in the future.

**Pull request closed without merging**